benchmark/pgn_benchmark.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:benchmark/benchmark.dart';
```

Comment:
rename file to `benchmark.dart` since it contains other benchmarks.

Also it would be nice to have a benchmark for `makePgn`